doc/admin.md

```diff
@@ -0,0 +1,565 @@
+# Administration
+
+-   [Configuration](#configuration)
+    -   [Configuration parameters:](#configuration-parameters)
+        -   [loglevel](#loglevel)
+        -   [contextBroker](#contextbroker)
+        -   [server](#server)
+        -   [stats](#stats)
+        -   [authentication](#authentication)
+        -   [deviceRegistry](#deviceregistry)
+        -   [mongodb](#mongodb)
+        -   [iotManager](#iotmanager)
+        -   [types](#types)
+        -   [eventType](#eventtype)
+        -   [service](#service)
+        -   [subservice](#subservice)
+        -   [providerUrl](#providerurl)
+        -   [iotaVersion](#iotaversion)
+        -   [appendMode](#appendmode)
+        -   [dieOnUnexpectedError](#dieonunexpectederror)
+        -   [singleConfigurationMode](#singleconfigurationmode)
+        -   [timestamp](#timestamp)
+        -   [defaultResource](#defaultresource)
+        -   [defaultKey](#defaultkey)
+        -   [componentName](#componentname)
+        -   [pollingExpiration](#pollingexpiration)
+        -   [pollingDaemonFrequency](#pollingdaemonfrequency)
+        -   [autocast](#autocast)
+        -   [multiCore](#multicore)
+        -   [fallbackTenant](#fallbacktenant)
+        -   [fallbackPath](#fallbackpath)
+        -   [explicitAttrs](#explicitattrs)
+        -   [defaultEntityNameConjunction](#defaultentitynameconjunction)
+        -   [relaxTemplateValidation](#relaxtemplatevalidation)
+    -   [Configuration using environment variables](#configuration-using-environment-variables)
+-   [Logs](#logs)
+    -   [Errors](#errors)
+-   [Alarms](#alarms)
+
+The **IoT Agent node library** is not a standalone product and should be added as a dependency to `package.json` of the
+project is going to be used in. The library is published in the NPM repository, so it can be added to the project by
+adding the following line to the `package.json` file:
+
+```json
+...
+"dependencies": {
+    "iotagent-node-lib": "x.y.z",
+}
+```
+
```

Comment:
```suggestion

where `x.y.z` is a particular version of the library (of the ones included [here](https://www.npmjs.com/package/iotagent-node-lib?activeTab=versions))

```

In addition, the following fragment could be include (I cannot use it in the suggestion due to the usage of ```):

---------

As alternative, you can use the master branch as dependency. In this case, you will be using the latest version of the code but note that some instability could exist (as the code in master is work in progress until the next version is closed).

```json
...
"dependencies": {
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
}
```

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1369/commits/6a9bfa02a72deba33617399dd87bd145429a8c18